e2e/visual/empty-ui.spec.js

```diff
@@ -1,6 +1,6 @@
-const { test, expect } = require('@playwright/test');
+import { test, expect } from '@playwright/test';
```

Comment:
@pinussilvestrus Sort of unrelated, but I decided to fully migrate to ES imports here since our node version supports it already